src/App/App.tsx

```diff
@@ -0,0 +1,49 @@
+import { useContext, useState } from "react";
+import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
+import { IntlProvider } from "react-intl";
+import Navbar from "../components/Navbar";
+import { Wrapper } from "./App.styles";
+import MainPage from "../pages/MainPage";
+import {
+  DarkModeContext,
+  DarkModeContextProvider,
+} from "../contexts/DarkModeContext";
+import TodoPage from "../pages/TodoPage";
+import UserTodos from "../pages/UserTodos";
+import { messages } from "./App.const";
+
+function App() {
+  const [search, setSearch] = useState("");
+  const [language, setLanguage] = useState<string>("en");
+  const { isDarkMode } = useContext(DarkModeContext);
+  return (
+    <DarkModeContextProvider>
+      <IntlProvider
+        messages={messages[language as keyof typeof messages]}
```

Comment:
a nie zadziała tutaj po prostu:
 messages={messages[language]}

Reply:
Dostaję wtedy taki error:
Element implicitly has an 'any' type because expression of type 'string' can't be used to index type

Reply:
oki, racja :D 
można to też obejść dodając typ w tym obiekcie 'messages', ale tak też jest git